docs/tex/tutorials/inference-networks.tex

```diff
@@ -75,10 +75,10 @@ \subsubsection{Implementation}
 x_ph = tf.placeholder(tf.float32, [M, 28 * 28])
 \end{lstlisting}
 
-We build a neural network using Keras.
-It takes a 28 by 28 pixel image and outputs a $10$-dimensional
-output, one for the mean (\texttt{mu}) and one for the standard
-deviation (\texttt{sigma}).
+We build a neural network using Keras consisting of two dense layers, one for the mean (\texttt{loc}) and one for the standard
+deviation (\texttt{scale}).
+Each dense layer takes a 28 by 28 pixel image and outputs $10$ values, one for each digit class.
```

Comment:
replace "Each dense layer" with "The network"

Reply:
Ok. Also, would it be more correct to say "The network...outputs 20 values (10 means and 10 standard deviations)" instead?

Reason: I'm not sure how "network" is defined here. Does the network include the dense layers only or does the network include both the dense layers and the normal?